packages/eas-cli/src/metadata/utils/date.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Remove time precision from a date to avoid potential errors with the App Store.
+ *
+ * "status": "409",
+ * "code": "ENTITY_ERROR.ATTRIBUTE.INVALID",
+ * "title": "An attribute value is invalid.",
+ * "detail": "The attribute 'earliestReleaseDate' only allows hour precision",
+ * "source": {
+ *   "pointer": "/data/attributes/earliestReleaseDate"
+ * }
+ */
+export function removeDatePrecision(date: any): null | Date {
```

Comment:
This shouldn't need any type

Reply:
Fixed in https://github.com/expo/eas-cli/pull/1147